heartbeat.go

```diff
@@ -60,23 +60,17 @@ func (k *Kite) kontrolFunc(fn kontrolFunc) error {
 // use it during app initializations. After the registration a reconnect is
 // automatically handled inside the RegisterHTTP method.
 func (k *Kite) RegisterHTTPForever(kiteURL *url.URL) {
-	interval := time.NewTicker(kontrolRetryDuration)
-	defer interval.Stop()
+	for attempts := 0; ; attempts++ {
+		backoff := k.Backoff(attempts, kontrolRetryDuration, 1*time.Minute)
```

Comment:
PR doesn't build because of this. Also we are already using backoff in client.go with https://github.com/cenkalti/backoff. There instead of implementing a custom one, use the one above.